source/docs/1.0/usage/Notification-Routing.md

```diff
@@ -1,15 +1,57 @@
 ## Notification routing
 
-This is how events are turned into notifications, and how notifications are routed to contacts:
+The graphic below shows how events are turned into
+notifications, and how notifications are routed to contacts as alerts. Further down, you
+can find a step-by-step description of the process.
 
 ![notification routing](/images/notification-routing.gif)
 
-#### Creating the above animation
+## Details
 
-1. Export [architecture diagrams](/images/FlapjackArchitecture.key) to PNG
-2. Run this shell:
+1. **Receive events.**
+    * Events are generated by external check execution systems and created as json objects in the `events` queue in Redis.
+    * Processor removes events off the queue and, astonishingly, processes them one by one.
+
+1. **Determine when to generate a notification**. The event is discarded by the filters if ANY of the following are true:
+     * check is OK and it's the first event we've seen for this check (new check)
+     * check is OK and previous state was OK
+     * check is in scheduled or unscheduled maintenance
+     * check is failing and duration of current failure is less than the `initial_failure_delay` (default 30 seconds, can be specified in the incoming event)
+     * check is failing and time elapsed since last notification is less than `repeat_failure_delay` (default 60 sec, can be specified in the incoming event)
+
+2. **Find interested contacts**
```

Comment:
Are you suggesting that alert recipients are a disinterested lot? We use 'interested' a fair bit. It's in the gif, for eg.